doc/releases/changelog-dev.md

```diff
@@ -4,6 +4,36 @@
 
 <h3>New features since last release</h3>
 
+* A new operation `SpecialUnitary` was added, providing access to an arbitrary
+  unitary gate via a parametrization in the Pauli basis.
+  [(#3650)](https://github.com/PennyLaneAI/pennylane/pull/3650)
+ 
+  The new operation takes a single argument, a one-dimensional `tensor_like`
+  of length `4**num_wires-1`, where `num_wires` is the number of wires the unitary acts on.
```

Comment:
Might be nice to add something here about what theta is, maybe along the lines of what you included in the docstring:

```
The parameter ``theta`` refer to all Pauli words (except for the identity) in lexicographical 
order, which looks like the following for one and two qubits:

>>> qml.ops.qubit.matrix_ops.pauli_words(1)
['X', 'Y', 'Z']
>>> qml.ops.qubit.matrix_ops.pauli_words(2)
['IX', 'IY', 'IZ', 'XI', 'XX', 'XY', 'XZ', 'YI', 'YX', 'YY', 'YZ', 'ZI', 'ZX', 'ZY', 'ZZ']
```

I found this very helpful.